draco/asp/examples/binned_histogram.lp

```diff
@@ -0,0 +1,28 @@
+attribute(number_rows,root,100).
```

Comment:
I actually anticipated that the examples would be incomplete specs. We should probably separate complete from incomplete specs. For complete specs, you won't see much in terms of grounding.